here-naksha-app-service/src/main/resources/swagger/openapi.yaml

```diff
@@ -655,6 +655,33 @@ paths:
           $ref: '#/components/responses/ErrorResponse504'
         '513':
           $ref: '#/components/responses/ErrorResponse513'
+  '/hub/spaces/{spaceId}/search':
+    get:
+      tags:
+        - Read Features
+      summary: Search for features
+      description: >-
+        Searches for features in the space. The results are unordered and the
+        request does not allow continuation of the search, which is the main
+        difference when compared to the _iterate_ request.
```

Comment:
let's not mention anything for now about _iterate_ API, as it is not part of contract for now ... and we are yet to identify what can be supported in iterate and how ... so first statement is fine (about unordered result), but second statement (about _iterate_) can be skipped to avoid confusion.

Reply:
done